### DISCUSSION TITLE
Can I pull my favorites or collections?

### DISCUSSION THREAD

**Discussion by drdougtheman** [davidteather/TikTok-Api#924] [Q&A] [answered]
Is it possible to pull my favorites or my collections?

**Comment by ctdgunner** [comment, marked as answer]
So the TikTok API provides information to anonymous users. To access The Favorites or collections, you need to be logged in to your account. This could only be possible if all of TikTok's session information was stored in cookies that the python wrapped tiktok api could access. Alot would need to be done before this would be possible.

**Reply by ctdgunner** [reply]
This no doubt, would be very useful however. (I have already switched all of my favorited videos over to liked) took lots of swiping and tapping, but I got it done....

**Reply by drdougtheman** [reply]
that's a lot of work!
